examples/nextjs-typescript/components/Layout.tsx

```diff
@@ -1,41 +1,51 @@
 import React, { ReactNode } from "react"
-import Link from "next/link"
 import Head from "next/head"
+import { Container, Flex, Heading, HStack } from "@chakra-ui/core"
+import { ColorModeSwitcher } from "./ColorModeSwitcher"
+import { Logo } from "./Logo"
+import { NextChakraLink } from "./NextChakraLink"
 
 type Props = {
   children?: ReactNode
   title?: string
 }
 
-const Layout = ({ children, title = "This is the default title" }: Props) => (
+export const Layout = ({
+  children,
+  title = "This is the default title",
+}: Props) => (
   <div>
     <Head>
       <title>{title}</title>
       <meta charSet="utf-8" />
       <meta name="viewport" content="initial-scale=1.0, width=device-width" />
     </Head>
-    <header>
-      <nav>
-        <Link href="/">
-          <a>Home</a>
-        </Link>{" "}
-        |{" "}
-        <Link href="/about">
-          <a>About</a>
-        </Link>{" "}
-        |{" "}
-        <Link href="/users">
-          <a>Users List</a>
-        </Link>{" "}
-        | <a href="/api/users">Users API</a>
-      </nav>
-    </header>
-    {children}
-    <footer>
-      <hr />
-      <span>I'm here to stay (Footer)</span>
-    </footer>
+    <Container maxWidth="1200px">
+      <header>
+        <Flex py={4} justifyContent="space-between" alignItems="center" mb={8}>
+          <Flex justifyContent="space-between" alignItems="center">
+            <nav>
+              <HStack spacing={12}>
+                <NextChakraLink
+                  href="/"
+                  passHref={true}
```

Comment:
Shouldn't need to set this since `passHref` is hardcoded to `true` for `NextLink` in `NextChakraLink.tsx`.